src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,46 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import java.util.List;
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.academy");
+
     public static void main(String[] args) {
+        BookDao bookDao = (BookDao) injector.getInstance(BookDao.class);
+        List<Book> books = List.of(
+                new Book(null, "Book1", new BigDecimal(100)),
+                new Book(null, "Book2", new BigDecimal(150)),
+                new Book(null, "Book3", new BigDecimal(50))
+        );
+
+        books.forEach(bookDao::create);
+        System.out.println("Created books:");
+        books.forEach(System.out::println);
+        System.out.println();
+
+        Long bookId = 2L;
+        System.out.println("Book with id = [" + bookId + "]");
+        bookDao.findById(bookId).ifPresent(System.out::println);
+        System.out.println();
+
+        System.out.println("All books in the table:");
+        bookDao.findAll().forEach(System.out::println);
+        System.out.println();
+
+        Book anyBook = books.stream().findAny().orElseThrow();
```

Comment:
Using `findAny()` without any filtering or additional context is non-deterministic and may lead to inconsistent behavior during different runs of the application. If the goal is to update a random book, consider adding a comment to clarify this choice. Otherwise, use a more deterministic approach, such as selecting the first element or using a specific criterion for selection.